src/index.js

```diff
@@ -168,18 +172,57 @@ function showGame() {
   updatePlayerScore();
 }
 
-
 function updatePlayerScore() {
-  player1Score.innerText = `${player1.score}`
-  player2Score.innerText = `${player2.score}`
-  player3Score.innerText = `${player3.score}`
+  player1Score.innerText = `${players[0].score}`
+  player2Score.innerText = `${players[1].score}`
+  player3Score.innerText = `${players[2].score}`
 }
 
 function displaySelectedClue(event) {
   let clickedCard = event.target.closest(".clue-card");
-  let selectedClue = clueInfo.find(clue => clue.id == clickedCard.id)
+  selectedClue = clueInfo.find(clue => clue.id == clickedCard.id)
   let selectedCategory = clueCategories.find(category => category.id === selectedClue.categoryId)
   $('.selected-clue-category').text(`${selectedCategory.category.split(/(?=[A-Z])/).join(" ").toUpperCase()}`);
   $('.selected-clue-points').text(`${selectedClue.pointValue}`);
   $('.question').text(`${selectedClue.question}`);
 }
+
+function evaluateGuess() {
+  let response;
+  $('.answer-response').css("display", "flex");
+  if ($(".player-guess").val().toUpperCase() === selectedClue.answer.toUpperCase()) {
+    $(".response").text(`Correct! \n You get ${selectedClue.pointValue} points!`);
+    response = "correct";
+  } else {
+    $(".response").text(`Incorrect! \n The answer is ${selectedClue.answer}. \n You lose ${selectedClue.pointValue} points!`)
+    response = "incorrect";
+  }
+  $(".player-guess").val('');
+  calculateScore(response);
+}
+
+function calculateScore(response) {
+  let currentPlayer = players.find(player => player.turn);
```

Comment:
This method using .find( ) is great! I can now see why you placed that method of this.turn = !this.turn which at first did not make sense but I can see that it rotates the turns around with the array of players.